relayer/relays/beacon/message/message.go

```diff
@@ -39,12 +39,12 @@ func New(ctx context.Context, eg *errgroup.Group, writer *writer.ParachainWriter
 }
 
 func (m *Message) SyncBasic(ctx context.Context, eg *errgroup.Group, blockNumber <-chan uint64) error {
-	lastVerifiedBlockNumber, err := m.writer.GetLastBasicChannelMessage()
+	lastVerifiedBlockNumber, err := m.writer.GetLastBasicChannelBlockNumber()
 	if err != nil {
 		return fmt.Errorf("fetch last basic channel message block number: %w", err)
 	}
 
-	addressNonceMap, err := m.writer.GetLastBasicChannelNoncesByAddresses(m.addresses)
+	addressNonceMap, err := m.writer.GetLastBasicChannelNonceByAddresses(m.addresses)
```

Comment:
Now I'm thinking `GetLastBasicChannelNonceMap` and `GetLastBasicChannelNonce` might be better 🙈 Maybe let's leave this for now